projects/python-examples/algokit_utils_py_examples/transactions/atomic_transaction_groups.py

```diff
@@ -0,0 +1,82 @@
+from algokit_utils import AlgoAmount, PaymentParams
+
+from algokit_utils_py_examples.helpers import setup_localnet_environment
+
+
+def atomic_transaction_groups() -> None:
+    # example: ATOMIC_TRANSACTION_GROUP
+
+    algorand_client, dispenser, account1, account2, account3 = (
+        setup_localnet_environment()
+    )
+
+    """
+    Create a transaction group that will execute atomically
+    Either all transactions succeed, or they all fail
+    """
+    algorand_client.new_group().add_payment(  # First transaction: Payment from A to B
```

Comment:
Sorry - I changed my accounts from account1, account2 to accountA, accountB ... felt it was a bit clearer. If you agree, I'd change the var names from localnet setup. Otherwise, update the comments referencing account A/B to 1/2.